src/ui/date/cell/DateCell.tsx

```
@@ -0,0 +1,44 @@
+  iso,
+  isDisabled,
+  isSelected,
+  isToday,
+  handleSelect,
+}: DateCellProps) {
+  const [y, m, d] = iso.split('-');
+  const srLabel = `${y}년 ${Number(m)}월 ${Number(d)}일`;
+  return (
+    <div
+      role='gridcell'
+      aria-selected={isSelected}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<td>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)